components/StoryblokProvider.tsx

```diff
@@ -57,15 +57,42 @@ const components = {
   sbWysiwyg: SbWysiwyg,
 };
 
-storyblokInit({
-  accessToken: process.env.STORYBLOK_ACCESS_TOKEN,
-  use: [apiPlugin],
-  apiOptions: {
-    region: 'us',
-  },
-  components,
-});
+interface ProviderProps {
+  children: React.ReactNode;
+  isEditor?: boolean;
+};
+
+export default function StoryblokProvider({ children, isEditor = false }: ProviderProps) {
+
+  let accessToken = ''; // No access token because this is in client side code.
+  if (isEditor) {
+    if (typeof window !== 'undefined') {
+      const urlParams = new URLSearchParams(window.location.search);
+      accessToken = urlParams.get('token') || accessToken;
+    }
+  }
+  // Temporarily override console.error to squeltch errors from Storyblok.
+  // Storyblok Init wants an api key but I don't want it in the client side code nor do I want to fetch from
+  // Storyblok's api on the front end.
+  const originalConsoleError = console.error;
+  console.error = () => {};
```

Comment:
There is a `console.error` buried in the StoryblokInit function that is showing up on the browser console when there is no API key. This kills it.